### ISSUE TITLE
Cannot uninstall this plugin.

### ISSUE THREAD

**Issue by fishfree** [LafColITS/Moodle-auth_casattras#38]
I firstly disabled this plugin, but the uninstall link didn't appear as below:
![image](https://user-images.githubusercontent.com/1741341/205649750-50dea93c-230f-49d3-b920-24267e51f554.png)

Then I tried to uninstall directly by the URL: /admin/plugins.php?uninstall=auth_casattras&confirm=0&return=manage The page showed  error as below:
![image](https://user-images.githubusercontent.com/1741341/205649975-b8b58e76-4e90-499d-b1e8-dbeaeaa115cf.png)

**Comment by mackensen** [comment]
@fishfree by default, an auth plugin won't let you uninstall if there are users who have it set as their auth method. Is this the case in your environment?

**Comment by fishfree** [comment]
@mackensen No. I had deleted all casattras users before tring to uninstall it. Thank you!
